codalab/model/bundle_model.py

```diff
@@ -1142,7 +1147,10 @@ def update_disk_metadata(self, bundle, bundle_location, enforce_disk_quota=False
 
         # TODO(Ashwin): make this non-fs specific
         data_hash = '0x%s' % (path_util.hash_directory(bundle_location, dirs_and_files))
-        data_size = path_util.get_size(bundle_location, dirs_and_files)
+        if hasattr(bundle.metadata, 'data_size'):
```

Comment:
what does this change do? does it fix a separate bug?

Reply:
It's not a bug, the data size was just being recomputed in this step so I skip the recomputation. I can add it back; it was just an optimization